### PR TITLE
Feature/241

### PR DESCRIPTION
**Pull request by mefaba**
This is a draft request. The reason is I want to be sure that I am heading to right way structurally. For example,
1-Is it okay to use Dialog component for this cards. 
2-Is it okay to use a conditional rendering for DriverFullProfile, RequestPage(component), Backdrop(Loading Component) and SnackBar(success component)
#241 

There might be also many typo since I did not fully checked the code.

### CODE REVIEW

File: web-client/src/shared/components/drivers-search/List/DriverCardVT/index.tsx [hunk 1]
```diff
@@ -2,39 +2,60 @@ import React, { useState } from 'react';
 import { Driver } from '../../../../redux/driversSearch/index';
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-search/List/DriverCardVT/index.tsx:2]: This looks good to me

File: web-client/src/shared/components/drivers-search/List/DriverFullProfile/RequestPage/index.tsx [hunk 2]
```diff
@@ -1,11 +1,85 @@
-import React from 'react';
+import React, { HtmlHTMLAttributes, useState } from 'react';
```
Comment (l2silver) [inline web-client/src/shared/components/drivers-search/List/DriverFullProfile/RequestPage/index.tsx:1]: Ah ok, so here we're look to take advantage of the yup and react-hook-form library that we're using throughout the site. Can you try implementing this page using those libraries. There should be examples in our code base for you to follow.

**Review by l2silver — CHANGES_REQUESTED** [review]
So this is pretty close to being done. Definitely on the right path, just need to try to use the react-hook-form library for the request page. Awesome work!

File: web-client/src/shared/components/drivers-search/List/DriverCardVT/index.tsx (hunk 1 above)
Comment (mefaba) [reply web-client/src/shared/components/drivers-search/List/DriverCardVT/index.tsx:2]: Thanks

File: web-client/src/shared/components/drivers-search/List/DriverFullProfile/RequestPage/index.tsx (hunk 2 above)
Comment (mefaba) [reply web-client/src/shared/components/drivers-search/List/DriverFullProfile/RequestPage/index.tsx:1]: Okay, I will look documentation of yup, react-hook-form and  also I will look implemented pages/components in our repo.

**Review by l2silver — APPROVED** [review]
Excellent work!

**Comment by l2silver** [comment]
@mefaba  One thing though. Can you fix the travis ci error

**Review by WendyChenj — APPROVED** [review]

**Comment by mefaba** [comment]
> @mefaba One thing though. Can you fix the travis ci error

fixed travis ci errors

**Pull request merged**